pkg/config/dynamic/http_config.go

```diff
@@ -293,13 +293,17 @@ type Server struct {
 	// Scheme can only be defined with label Providers.
 	Scheme string `json:"-" toml:"-" yaml:"-" file:"-" kv:"-"`
 	Port   string `json:"-" toml:"-" yaml:"-" file:"-" kv:"-"`
-	// HealthCheck enables passive health checks for this server
+	// HealthCheck enables passive health checks for this server.
 	HealthCheck *PassiveHealthCheck `json:"healthCheck,omitempty" toml:"healthCheck,omitempty" yaml:"healthCheck,omitempty" export:"true"`
 }
 
+// +k8s:deepcopy-gen=true
+
 type PassiveHealthCheck struct {
+	// FailTimeout sets the time during which a number of failed attempts must happen for the server to be marked unavailable.
 	FailTimeout ptypes.Duration `json:"failTimeout,omitempty" toml:"failTimeout,omitempty" yaml:"failTimeout,omitempty" export:"true"`
-	MaxFails    int             `json:"maxFails,omitempty" toml:"maxFails,omitempty" yaml:"maxFails,omitempty" export:"true"`
+	// MaxFails sets the number of failed attempts that must occur during the fail timeout.
+	MaxFails int `json:"maxFails,omitempty" toml:"maxFails,omitempty" yaml:"maxFails,omitempty" export:"true"`
```

Comment:
`MaxFails` is slightly ambiguous to me.
WDYT about replacing it with `FailureThreshold` (or `MaxFailedAttempts`) to improve clarity, making it easier to understand that it sets the limit of allowed failures within the failure window before a server is marked as unhealthy?

Reply:
I changed to MaxFailedAttempts